encoding.bs

```diff
@@ -340,9 +346,9 @@ happening.
    <dd><a>Push</a> U+FFFD to <var>output</var>.
 
    <dt>"<code>html</code>"
-   <dd><a>Prepend</a> U+0026, U+0023, followed by the
-   shortest sequence of <a>ASCII digits</a> representing <var>result</var>'s
-   <a>code point</a> in base ten, followed by U+003B to <var>input</var>.
+   <dd><a>Push</a> the result of <a>UTF-8 encoding</a> U+0026 (&amp;), U+0023 (#), followed by the
```

Comment:
I suggest saying "0x26 (& encoded as UTF-8), 0x23 (# encoded as UTF-8)" and then "0x3B (; encoded as UTF-8)" to remove a layer of indirection.

Reply:
The problem is that we don't have a definition of https://infra.spec.whatwg.org/#ascii-digit that's byte-oriented. I could inline one, "the shortest sequence of 0x30 (0) to 0x39 (9), inclusive, representing ... in base ten?" if that's preferable. I guess this is also a good moment to start using https://infra.spec.whatwg.org/#code-point-value.

Reply:
I'd prefer inlining such a definition over talking about UTF-8 encode here.

Reply:
Gave that a go so you can see how it looks.